<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>obfCoder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>631</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>241</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="cb_deleteComments">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>209</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete comments</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_changeLoops">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>209</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Change loops</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_ChangeVariables">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>209</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rename variables</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QSpinBox" name="sp_AmountVariables">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sp_AmountCodes">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of variables</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of methods per variable</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_AddJunk">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>209</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Additional code</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerSum">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sum</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerSub">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Difference</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerMul">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiply</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerConnected">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Loop For connected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerSemiConnected">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Loop For semi-connected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerNonConnected">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Loop For unconnected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_JunkerInc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Incrementation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cb_DeleteEnters">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete transitions to new line</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="cb_Encryption">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>209</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Encryption</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_EncToFile">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>To file</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_EncOnlyFor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Only loops for</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cb_DeleteSpace">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete unnecessary spaces</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_LoadFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabelIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabelOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_SaveFile">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>12</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_openFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
